pyproject.toml

```diff
@@ -26,6 +26,9 @@ exclude = [
     "**/*.pyi",
 ]
 
+[tool.poetry.urls]
+"Bug Tracker" = "https://github.com/python-poetry/poetry/issues"
+
```

Comment:
I think we should also add a paragraph to the `README.md`, that issues belonging to `poetry-core` must be reported in the issue tracker of `poetry`.

Reply:
That's coming later in the contributing md file.

Reply:
Ok, I see that we already send people to the poetry issue tracker, when they try to open an issue in poetry-core. So I think that's fine.